lib/primitives/claim.js

```
@@ -194,35 +276,70 @@ class Claim extends bio.Struct {
     this.blob = proof.encode();
     return this;
   }
 
+  /**
+   * Instantiate claim from raw proof.
+   * @param {Buffer} blob
+   * @returns {Claim}
+   */
+
   static fromBlob(blob) {
     return new this().fromBlob(blob);
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)